{{cookiecutter.project_slug}}/clients/mobile/react-native/src/services/user/hooks.ts

```diff
@@ -6,28 +6,21 @@
 import { useEffect } from 'react'
 import { useQuery, useMutation } from '@tanstack/react-query'
 import { useAuth } from '@stores/auth'
-import { userApi } from './api'
+import { userApi, userQueries } from '.'
```

Comment:
do not import from `.` it sometimes fails to resolve depending on the importing order and module resolution

Even though this is done by VS code automatically check that when you import from the same folder reference the file and not the index